package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "csv-component-oih",
+  "description": "Blended Edge CSV Component",
+  "scripts": {
+    "pretest": "eslint lib/ spec --fix",
+    "test": "nyc mocha spec",
+    "integration-test": "mocha spec-integration",
+    "compile": "eslint . --ext .ts && tsc"
+  },
+  "engines": {
+    "node": "14"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/elasticio/csv-component-oih.git"
+  },
+  "keywords": [
+    "csv",
+    "OIH",
+    "ipaas",
+    "connector"
+  ],
+  "author": "Blended Edge",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/blendededge/csv-component-oih/issues"
+  },
+  "homepage": "https://github.com/blendededge/csv-component-oih#readme",
+  "dependencies": {
+    "@blendededge/ferryman-extensions": "^1.1.2",
+    "@elastic.io/component-commons-library": "1.1.5",
+    "@openintegrationhub/ferryman": "^1.7.0",
+    "@types/papaparse": "^5.3.2",
+    "axios": "^0.21.4",
+    "elasticio-node": "0.0.9",
+    "elasticio-sailor-nodejs": "^2.6.7",
```

Comment:
I see the start.sh but not all the dependencies are here. Being that this is a new component I'm fine with not being backwards compatible to support the new environment variables.  It should just be completely correct either way.

Reply:
Yes- you do need two versions of ferryman, but only one version of sailor. I installed the second version of ferryman under `ferryman-1-7-0` instead of prefixing with `@openintegrationhub` which is probably why you didn't see it before. I have changed the name.